app/helpers/static_pages_helper.rb

```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```

Comment:
bỏ file k dùng
kill. toàn sai lỗi cơ bản thế e

Reply:
em vội vàng xuân diệu quá ạ =(( em sẽ cẩn thận hơn